nss-day-cohort-46/nss-day-cohort-46.github.io
# Mp background

## Link to Ticket

TITLE [Link]()

## Description of Proposed Changes
- Proposed background image change. It is the texture of the "normal type" pokemon card. 
- The colors are also changed to gold.

## Steps to Test
- run the following code in command line
- look at the background image and see what you think

```
git fetch --all
git checkout mp-background

```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* main styling

## Mentions @username

Tag users that need to review this code
